app/javascript/pages/getAssistance/document-checklist.tsx

```diff
@@ -1,19 +1,161 @@
-import { t } from "@bloom-housing/ui-components"
+import { ContentAccordion, t } from "@bloom-housing/ui-components"
 import React from "react"
 import AssistanceLayout from "../../layouts/AssistanceLayout"
 import withAppSetup from "../../layouts/withAppSetup"
+import { renderInlineMarkup } from "../../util/languageUtil"
 
 const DocumentChecklist = () => {
+  const preferences = [
+    {
+      title: t("listings.lotteryPreference.Neighborhood Resident Housing Preference (NRHP).title"),
+      expanded: (
```

Comment:
Overall, very readable code and easy to understand! I am curious to hear your thoughts on creating a component for the expanded field, since there is a lot of similar structure between each expanded preference. This is just an idea though, I think either approach works great.